File: python5TimesPractice/longestString.py



def sample(str):
    arr = str.split(" ")
    print(arr)
   
    longestWord = ""
    for word in arr:
        if len(word) > len(longestWord):
            longestWord = word
    
    print(longestWord)


sample("my name is prashant kamble")

File: javascript5TimesPractice/longestString.js
// 1 = find the longest word from the string

function sample(str){
    arr =  str.split(" ")

    var longestWord = ""
    for(i=0; i < arr.length; i++){
        if(arr[i].length > longestWord.length){
            longestWord = arr[i]
        }
    }
    console.log(longestWord)
}

sample("my name is prashant kamble")



